components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -5057,6 +5059,12 @@ private APIDTO importAsyncAPISpecification(InputStream definition, String defini
             if (isServiceAPI) {
                 apiDTOFromProperties.setType(PublisherCommonUtils.getAPIType(service.getDefinitionType(), protocol));
             }
+            if (SolaceConstants.SOLACE_ENVIRONMENT.equals(apiDTOFromProperties.getGatewayVendor())) {
+                apiDTOFromProperties.getPolicies().add("AsyncUnlimited");
```

Comment:
is it make sense add these as constants ?

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10842/commits/c41ba333ee522fd29e341c130570450703f787bf